mapzen-android-sdk/build.gradle

```diff
@@ -81,7 +81,7 @@ task verify(dependsOn: ['compileDebugSources',
                         'lint'])
 
 dependencies {
-  compile 'com.mapzen:mapzen-core:0.0.1-SNAPSHOT'
```

Comment:
Up until now `core` and `places` were using version 0.0.1-SNAPSHOT and `sdk` was using version 1.3.0-SNAPSHOT (see core/gradle.properties). This pull request syncs all modules to use version 1.3.0-SNAPSHOT.